streaming/src/main/scala/org/apache/spark/streaming/receiver/ReceivedBlockHandler.scala

```diff
@@ -51,7 +54,8 @@ private[streaming] trait ReceivedBlockHandler {
  * that stores the metadata related to storage of blocks using
  * [[org.apache.spark.streaming.receiver.BlockManagerBasedBlockHandler]]
  */
-private[streaming] case class BlockManagerBasedStoreResult(blockId: StreamBlockId)
+private[streaming] case class BlockManagerBasedStoreResult(blockId: StreamBlockId,
```

Comment:
nit (only if there are other changes): can you move both params to the next line.